jupyter-book/introduction/raw_data_processing.md

```diff
@@ -455,94 +454,208 @@ To do this, you can replace the above commands with the following (instructions
+simpleaf quant \
+-c 10xv3 -t 8 \
+-1 $reads1 -2 $reads2 \
+-i simpleaf_index/index \
+-u -r cr-like \
+-m simpleaf_index/index/t2g_3col.tsv \
+-o simpleaf_quant
+```
+
+After running these commands, the resulting quantification information can be found in the `simpleaf_quant/af_quant/alevin` folder. Within this directory, there are three files: `quants_mat.mtx`, `quants_mat_cols.txt`, and `quants_mat_rows.txt`, which correspond, respectively, to the count matrix, the gene names for each column of this matrix, and the corrected, filtered cell barcodes for each row of this matrix. The tail lines of these files are shown below. Of note here is the fact that `alevin-fry` was run in the USA-mode (<u>u</u>nspliced, <u>s</u>pliced, and <u>a</u>mbiguous mode), and so quantification was performed for both the spliced and unspliced status of each gene — the resulting `quants_mat_cols.txt` file will then have a number of rows equal to 3 times the number of annotated genes which correspond, to the names used for the spliced (S), unspliced (U), and splicing-ambiguous variants (A) of each gene. 
+
+```bash
+# Each line in `quants_mat.mtx` represents
+# a non-zero entry in the format row column entry
+$ tail -3 simpleaf_quant/af_quant/alevin/quants_mat.mtx
+138 58 1
+139 9 1
+139 37 1
+
+# Each line in `quants_mat_cols.txt` is a splice status
+# of a gene in the format (gene name)-(splice status)
+$ tail -3 simpleaf_quant/af_quant/alevin/quants_mat_cols.txt
+ENSG00000120705-A
+ENSG00000198961-A
+ENSG00000245526-A
+
+# Each line in `quants_mat_rows.txt` is a corrected
+# (and, potentially, filtered) cell barcode
+$ tail -3 simpleaf_quant/af_quant/alevin/quants_mat_rows.txt
+TTCGATTTCTGAATCG
+TGCTCGTGTTCGAAGG
+ACTGTGAAGAAATTGC
+```
+
+We can load the count matrix into python as an [`AnnData`](https://anndata.readthedocs.io/en/latest/) object using the `load_fry` function from [`pyroe`](https://github.com/COMBINE-lab/pyroe). A similar function, [loadFry](https://rdrr.io/github/mikelove/fishpond/man/loadFry.html), has been implemented in the [`fishpond`](https://github.com/mikelove/fishpond) R package.
+
+```python
+import pyroe
+
+quant_dir = 'simpleaf_quant/af_quant'
+anndata = pyroe.load_fry(quant_dir)
+```
+
+The default behavior loads the `X` layer of the `anndata` object as the sum of the spliced and ambiguous counts for each gene. However, recent work{cite}`Pool2022` and [updated practices](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/release-notes) suggest that the inclusion of intronic counts, even in single-cell RNA-seq data, may increase sensitivity and benefit downstream analyses. While the best way to make use of this information is the subject of ongoing research, since `alevin-fry` automatically quantifies spliced, unspliced, and ambiguous reads in each sample, the count matrix containing the total counts for each gene can be simply obtained as follows:
```

Comment:
```suggestion
The default behavior loads the `X` layer of the `AnnData` object as the sum of the spliced and ambiguous counts for each gene. However, recent work{cite}`Pool2022` and [updated practices](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/release-notes) suggest that the inclusion of intronic counts, even in single-cell RNA-seq data, may increase sensitivity and benefit downstream analyses. While the best way to make use of this information is the subject of ongoing research, since `alevin-fry` automatically quantifies spliced, unspliced, and ambiguous reads in each sample, the count matrix containing the total counts for each gene can be simply obtained as follows:
```